css/style.css

```diff
@@ -2,7 +2,7 @@
 
 * {
   box-sizing: border-box;
-  font-family: "Poppins", sans-serif;
+  font-family: "Poppins", sans-serif, Arial, Helvetica;
 }
```

Comment:
kindly as was required to the project requirement You should stick to the design as much as possible (e.g. font, colors, images, tests, margins)